index.js

```diff
@@ -20,14 +20,6 @@ module.exports = function (a, b) {
 	var aLen = a.length;
 	var bLen = b.length;
 
-	if (aLen === 0) {
```

Comment:
That's right indeed. But I guess we should benchmark the change to see if this yields any positive result.